### PR TITLE
Audited and improved existing metadata for main docs pages

### LINKED ISSUE #131: Improve the metadata for each page
Specifically the `title` and `description`. We want better SEO and social media sharing.

In MDX, this is at the top of the file:

https://github.com/Qiskit/documentation/blob/c0b5533185785aab04c695525c81df7cd0792ebe/docs/support.mdx#L1-L3

In notebooks, it's in the file's `metadata` section, which is much easier to miss:

https://github.com/Qiskit/documentation/blob/c0b5533185785aab04c695525c81df7cd0792ebe/docs/build/backend_reset.ipynb#L770-L771

The title is used for the browser tab's title and for the SEO and social media title. Description is used for SEO and social media. Best practices to follow for the description:

* No more than 160 characters
* Contains relevant keywords to the page, especially keywords developers might be looking for. 
    * But also keyword stuffing negatively impacts SEO, like `Hadamon Gate, quantum computing, Qiskit, Runtime, transpile` is a bad description. It should read naturally. Reminder the description is used for rich previews on social media & the Google result.
* Unique per page. That's helpful for our end users too when they are looking in the search results on what page they want.
* Think about what is engaging for our users to want to open the link when they're on Google and social media. It's not only about Google showing our result up top; it's also that people want to click it.

### PR DESCRIPTION
**Pull request by javabster**
closes #131

### CODE REVIEW

**Review by Eric-Arellano — APPROVED** [review]
This is great! Thanks so much Abby.

File: docs/build/circuit-visualization.ipynb [hunk 1]
```diff
@@ -1301,7 +1301,7 @@
  "metadata": {
   "anaconda-cloud": {},
   "celltoolbar": "Tags",
-  "description": "Plotting data in Qiskit notebook",
+  "description": "Create visualisations of circuits and plot job data using the Qiskit visualization module ",
```
Comment (abbycross) [inline docs/build/circuit-visualization.ipynb:1304]: I love that we have both a British and an American version of the spelling here 😆 I wonder if it's useful to keep it as-is to turn up more in searches? Google must account for this kind of variety in spelling, right?

File: docs/build/circuit-library.ipynb [hunk 2]
```diff
@@ -430,7 +430,7 @@
  ],
  "metadata": {
   "title": "What is the circuit library?",
```
Comment (abbycross) [inline docs/build/circuit-library.ipynb:432]:
```suggestion
  "title": "Circuit library",
```
Matching the metadata with the page title

File: docs/run/instances.mdx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Instances
-description: What instances are and how to use them
+description: What IBM Quantum platform instances are and how to use them
```
Comment (abbycross) [inline docs/run/instances.mdx:3]:
```suggestion
description: What IBM Quantum Platform instances are and how to use them
```

File: docs/start/setup-channel.mdx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Select and set up an IBM Quantum channel
-description: Install and set up IBM Quantum Platform or IBM Quantum on IBM Cloud to submit Qiskit and Qiskit Runtime jobs
+description: Installation and set up instructions for IBM Quantum Platform or IBM Quantum on IBM Cloud to submit Qiskit and Qiskit Runtime jobs
```
Comment (abbycross) [inline docs/start/setup-channel.mdx:3]:
```suggestion
description: Installation and setup instructions for IBM Quantum Platform or IBM Quantum on IBM Cloud to submit Qiskit and Qiskit Runtime jobs
```

File: docs/build/circuit-visualization.ipynb (hunk 1 above)
Comment (Eric-Arellano) [reply docs/build/circuit-visualization.ipynb:1304]: IBM's style guide says to use American spelling hah

Comment (javabster) [reply docs/build/circuit-visualization.ipynb:1304]: omg you can really tell im going through an identity crisis 😆

Comment (Eric-Arellano) [reply docs/build/circuit-visualization.ipynb:1304]: 😅

Screenshot 2023-11-14 at 1 45 15 PM

**Review by abbycross — APPROVED** [review]
Ready to go once the metadata has been fixed to match the title in circuit-library (see https://github.com/Qiskit/documentation/pull/345#discussion_r1392767666), so approving!

**Pull request merged**